searchapp/models.py

```diff
@@ -2,28 +2,63 @@
 from django.contrib.auth.models import User
 
 
-
 class Mentor(models.Model):
     REGULAR = 1
     SUPERADMIN = 2
     MENTOR_TYPE_CHOICES = (
         (REGULAR, 'REGULAR'),
         (SUPERADMIN, 'SUPERADMIN'),
     )
-    username = models.CharField(max_length=40, unique=True, blank=False, null=False)
+    username = models.CharField(
+        max_length=40, unique=True, blank=False, null=False)
     password = models.CharField(max_length=40, blank=False, null=False)
     phone = models.CharField(max_length=40, blank=True, null=True)
-    email = models.CharField(max_length=40, unique=True, blank=False, null=False)
-    mentor_type = models.IntegerField(choices=MENTOR_TYPE_CHOICES, default=REGULAR)
+    email = models.CharField(
+        max_length=40, unique=True, blank=False, null=False)
+    mentor_type = models.IntegerField(
+        choices=MENTOR_TYPE_CHOICES, default=REGULAR)
     full_name = models.CharField(max_length=40)
-    created_time = models.DateTimeField(auto_now_add=True, blank=True, null=True)
+    created_time = models.DateTimeField(
+        auto_now_add=True, blank=True, null=True)
 
     def __str__(self):
         return self.username
 
 
+class Board(models.Model):
+    BOARD_TYPE_CHOICES = (
```

Comment:
Do not do this. Incase a new board needs to be added, this will force code changes. Ideally, you'd just want to be able to insert a new Board to the database and not require any code changes.

Reply:
This was to keep the naming and format of boards consistent in terms of capital and small letters and their abbreviations. I can change it if we do not need these checks.